scripts/packageAdBlock.js

```diff
@@ -54,6 +54,12 @@ const processComponent = (binary, endpoint, region, keyDir,
   const parsedManifest = util.parseManifest(originalManifest)
   const id = util.getIDFromBase64PublicKey(parsedManifest.key)
 
+  // TODO - previous download failures should prevent the attempt to package the component.
+  if (!fs.existsSync(originalManifest)) {
+    console.warn(`Missing manifest for ${componentSubdir}. Skipping.`)
```

Comment:
Shouldn't we still report this via Sentry?

Reply:
The only way this happens so far is if there was a previous failure reported by Sentry. Long-term, I wouldn't be comfortable leaving it this way... but in the short term before #844 (which is basically ready), there's no need to have duplicate Sentry notifications every time.